exercises/nucleotide-count/nucleotide_count.ml

```diff
@@ -0,0 +1,10 @@
+open Base
+(* open Base.Continue_or_stop *)
+
+let empty = Map.empty (module Char)
+
+let count_nucleotide s c =
+  raise (Failure "'count_nucleotide' is missing")
```

Comment:
You can use failwith I think

Reply:
Didn't know, thanks!

In Standard ML, `raise` is a special syntactic construct, so I'm allowed to write `raise Fail "Foo"`, but in OCaml, `raise` seems to be a regular function, so the parenthesis is necessary. I like OCaml's choice better.